source/_components/sensor.fail2ban.markdown

```diff
@@ -48,6 +49,11 @@ file_path:
   required: false
   type: string
   default: /var/log/fail2ban.log
+scan_interval:
```

Comment:
This is already documented for all [polling platforms](https://www.home-assistant.io/docs/configuration/platform_options/#scan-interval).

Remove it.